universidade-do-porto-faculdade-de-engenharia-chicago-pt.csl

```diff
@@ -0,0 +1,760 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago" default-locale="pt-PT">
+  <info>
+    <title>Universidade do Porto - Faculdade de Engenharia - Chicago Manual of Style - PT</title>
```

Comment:
```suggestion
    <title>Universidade do Porto - Faculdade de Engenharia - Chicago Manual of Style (Português - Portugal) </title>
```

Reply:
Hi Adam, I have made the alteration you have suggested but could it be? (or something similar):
Universidade do Porto - Faculdade de Engenharia - Chicago 17th (autor-data) (Português - Portugal)
There are other Chicago styles... and this is an author-date format....

Reply:
or:
Universidade do Porto - Faculdade de Engenharia - Chicago Manual of Style 17th (autor-data) (Português - Portugal)
is it too long?